blocklylib-core/src/main/java/com/google/blockly/android/WorkspaceFragment.java

```diff
@@ -56,6 +56,9 @@
     private WorkspaceView mWorkspaceView;
 
     private boolean mDrawGrid = true;
+    private Integer mGridColor = null;
```

Comment:
We can make these unboxed ints if we just refer to the defaults in WorkspaceGridRenderer.  Make the constants package scoped to make them visible.  That also allows us to sidestep the if statements (with a tiny amount of one-time over overhead).

Reply:
WorkspaceFragment and WorkspaceGridRenderer are in different packages. I can either move the defaults to the former or make the latter public.
Any preference?

Reply:
I've gone for making WorkspaceGridRenderer public, but happy to change it if you'd prefer.